### PR TITLE
 Allow to have a null center in position and add name

### PR DESCRIPTION
**Pull request by yohanboniface**
Ref #108 #76

### CODE REVIEW

File: ban/tests/http/test_position.py [hunk 1]
```diff
@@ -56,6 +88,21 @@ def test_cannot_create_position_without_invalid_kind(client):
...
+    assert not models.Position.select().count()
+    url = '/position'
+    data = {
+        "kind": models.Position.ENTRANCE,
+        "positioning": models.Position.IMAGERY,
+        "housenumber": housenumber.id,
+    }
+    resp = client.post(url, data)
+    assert resp.status == falcon.HTTP_422
+    assert 'center' in resp.json['errors']
```
Comment (davidbgk) [inline ban/tests/http/test_position.py]: Check `name` presence too?

File: ban/tests/http/test_position.py [hunk 2]
```diff
@@ -303,6 +350,21 @@ def test_patch_with_wrong_version_should_fail(client, url):
...
+    assert models.Position.select().count() == 1
+    uri = url('position-resource', identifier=position.id)
+    data = {
+        "version": 2,
+        "center": "",
+        "name": ""
+    }
+    resp = client.patch(uri, body=json.dumps(data))
+    assert resp.status == falcon.HTTP_422
+    assert "center" in resp.json['errors']
```
Comment (davidbgk) [inline ban/tests/http/test_position.py]: Inconsistency in simple/double quotes?

File: ban/core/resource.py [hunk 3]
```diff
@@ -41,9 +42,10 @@ def _validate_coerce(self, coerce, field, value):
     def validate(self, data, instance=None, **kwargs):
         self.instance = instance
         super().validate(data, **kwargs)
-        if ('version' in self.schema and instance
-           and not self.document.get('version')):
-            self._error('version', errors.ERROR_REQUIRED_FIELD)
+        if hasattr(self.model, 'validate'):
+            for key, message in self.model.validate(self, self.document,
+                                                    instance).items():
+                self._error(key, message)
```
Comment (davidbgk) [inline ban/core/resource.py]: You might want [to use a different signature](https://github.com/nicolaiarocci/cerberus/blob/master/cerberus/cerberus.py#L177):

```self._error(*[errors in self.model.validate(self, self.document, instance).items()])``` (pseudo-code)

**Comment by davidbgk** [comment]
r+wc

**Comment by yohanboniface** [comment]
Merged through #110.

**Pull request closed without merging**